pint_server/pint_db_migrate/versions/b381843009e0_trying_enums.py

```diff
@@ -0,0 +1,73 @@
+"""Trying Enums
+
+Revision ID: b381843009e0
+Revises: 528cb85d6ad3
+Create Date: 2021-07-30 00:15:20.918363
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+from sqlalchemy.schema import Sequence, CreateSequence
+
+# revision identifiers, used by Alembic.
+revision = 'b381843009e0'
+down_revision = '528cb85d6ad3'
+branch_labels = None
+depends_on = None
+
+tablenames = ['amazonservers', 'googleservers', 'microsoftservers']
+
+def upgrade():
+    # manual commands
+    for tablename in tablenames:
+        op.drop_constraint('%s_pkey' % (tablename), tablename, type_='primary')
+        op.execute(CreateSequence(Sequence('%s_id_seq' % (tablename))))
+
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('amazonservers', sa.Column(
+        'id', sa.Integer(), nullable=False, server_default=sa.text("nextval('amazonservers_id_seq'::regclass)"))
```

Comment:
The server default could be done in the model, but then it should be one `id` column per server model